apps/blockchain/lib/blockchain/block/holistic_validity.ex

```diff
@@ -82,7 +82,8 @@ defmodule Blockchain.Block.HolisticValidity do
       |> check_state_root_validity(child_block, block)
       |> check_ommers_hash_validity(child_block, block)
       |> check_transactions_root_validity(child_block, block)
-      |> check_receipts_root_validity(child_block, block)
+
+    # |> check_receipts_root_validity(child_block, block)
```

Comment:
I've disabled this check since transaction receipts are not implemented yet

Reply:
can you  add todo comment with related the  github  issue?

Reply:
No I just can't do this